app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
```

Comment:
Here is the new PR: https://github.com/projecthydra-labs/curation_concerns/pull/780